Index: chungbuk2021/test.html
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+><!DOCTYPE html>\r\n<html lang=\"en\">\r\n<head>\r\n    <meta charset=\"UTF-8\">\r\n    <title>Title</title>\r\n</head>\r\n<body>\r\n\r\n</body>\r\n</html>
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/chungbuk2021/test.html b/chungbuk2021/test.html
--- a/chungbuk2021/test.html	(revision ece9fb238fd5bc7bde3412a7f6258ec7e25d99c1)
+++ b/chungbuk2021/test.html	(date 1646635865828)
@@ -2,9 +2,11 @@
 <html lang="en">
 <head>
     <meta charset="UTF-8">
-    <title>Title</title>
+    <title>Test</title>
+    <link rel="stylesheet" href="./site/css/test.css">
+    <script src="./site/js/test.js"></script>
 </head>
 <body>
-
+    <div class="gunami">test</div>
 </body>
 </html>
\ No newline at end of file
diff --git a/chungbuk2021/site/css/test.css b/chungbuk2021/site/css/test.css
new file mode 100644
diff --git a/chungbuk2021/site/js/test.js b/chungbuk2021/site/js/test.js
new file mode 100644
